task19/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <script type="module" src="js/slider.js" defer></script>
+    <script type="module" src="js/news.js" defer></script>
```

Comment:
If a module is specified, defer is not necessary.
Because by specifying module already has an effect.
```suggestion
    <script type="module" src="js/slider.js"></script>
    <script type="module" src="js/news.js"></script>
```

[ref](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attributes)